### PR TITLE
add copy button to kv secrets masked input

### PR DESCRIPTION
**Pull request by madalynrose**
screen shot 2018-10-26 at 6 53 11 pm

### CODE REVIEW

File: ui/app/templates/components/masked-input.hbs [hunk 1]
```diff
@@ -15,6 +15,15 @@
...
 	{{/if}}
+  {{#if allowCopy}}
+    <CopyButton
+      @clipboardText={{value}}
+      @class="copy-button button"
+      data-test-copy-button
+    >
+      <ICon @glyph="copy" aria-hidden="true" @size=16 />
+    </CopyButton>
+  {{/if}}
```
Comment (meirish) [inline ui/app/templates/components/masked-input.hbs]: Do you remember why we put these after the input? I was looking at that the other day and couldn't remember haha.

Comment (madalynrose) [reply ui/app/templates/components/masked-input.hbs]: was just talking to Joshua about this. our guess is accessibility

File: ui/app/styles/components/masked-input.scss [hunk 2]
```diff
@@ -27,48 +27,71 @@
...
+  // &:active,
+  // &.is-active,
+  // &:focus,
+  // &.is-focused,
+  // &:hover,
+  // &:focus:not(:active) {
+  //   color: $blue;
+  //   border: 0;
+  //   box-shadow: none;
+  // }
```
Comment (meirish) [inline ui/app/styles/components/masked-input.scss]: do we still need the commented styles?

**Comment by madalynrose** [comment]
@meirish addressed the spacing with flexbox :success:

**Pull request merged**